src/index.html

```diff
@@ -9,10 +9,42 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
```

Comment:
The `href` attribute is linking to a SCSS file. Browsers cannot directly interpret SCSS, so ensure that this SCSS file is compiled to CSS and that the compiled CSS file is linked here instead.